Ensure init phase runs before configd

If user_data wants to drop in a completely new configuration, write_files
(which is part of the "init" phase) needs to run before configd.service
starts.

---

--- a/systemd/cloud-init.service.tmpl
+++ b/systemd/cloud-init.service.tmpl
@@ -32,6 +32,7 @@
 Conflicts=shutdown.target
 {% endif %}
 Before=systemd-user-sessions.service
+Before=configd.service
 ConditionKernelCommandLine=cloud-init
 # Triggered by IP auto-configuration
 ConditionPathExistsGlob=|/run/net-*.conf
@@ -40,7 +41,7 @@
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/sg vyattacfg "/usr/bin/cloud-init init"
+ExecStart=/usr/bin/cloud-init init
 RemainAfterExit=yes
 TimeoutSec=0
 
